src/test/regress/expected/columnar_alter_set_type.out

```diff
@@ -50,3 +50,25 @@ SELECT * FROM test_alter_table ORDER BY a;
 (4 rows)
 
 DROP TABLE test_alter_table;
+-- https://github.com/citusdata/citus/issues/5927
```

Comment:
Maybe also explain a bit what we are testing here. The link is nice for backreference, however, I'd like a small explanation here.

Something like
```sql
--  Make sure that the correct table options are used when rewriting the table.
-- This is reflected by the VACUUM VERBOSE output right after a rewrite showing
-- that all chunks are compressed with the configured compression algorithm
-- original bug: https://github.com/citusdata/citus/issues/5927
```